networking/network_observability/installing-operators.adoc

```diff
@@ -1,13 +1,27 @@
 :_content-type: ASSEMBLY
 [id="installing-network-observability-operators"]
-= Installing Network Observability operators in {product-title}
+= Installing the Network Observability operator
```

Comment:
Operator should be capitalized.

Reply:
The really dumb thing about me not capitalizing this is that I had multiple convos with people about this and then didn't uniformly capitalize in my own docs. Anyways, thanks for pointing all these out to me.